### ISSUE TITLE
Jazzy vs Ruby version

### ISSUE THREAD

**Issue by akac** [realm/jazzy#906]
So on my Mac with the stock ruby, Jazzy works 99.99% of the way. I have it nicely automated with fast lane as well.

On my co-worker's Mac, he has rvm installed and 6 different versions of Ruby since he's got a few tools that require newer Ruby versions.

On his Mac, jazzy does *not* work properly. Same paths, same folders, same .jazzy.yaml file and it won't find the source. So he uninstalled all the Ruby versions except for 2.41. It now does work…partly. It gets about 50% of the symbols. Here is his output:

```
[17:08:13]: ▸ Using config file /Users/amudubagilu/navigation-ios/.jazzy.yaml
[17:08:17]: ▸ /Users/amudubagilu/project-ios/Project/ProjectUmbrella.h:8:9: fatal error: could not build module ‘UIKit’
[17:08:44]: ▸ Found conflicting type declarations with the same name, which may indicate a build issue or a bug in Jazzy: function NS_ENUM, function NS_ENUM, function NS_ENUM, function NS_ENUM, function NS_ENUM, function NS_ENUM, function NS_ENUM, function NS_ENUM, function NS_ENUM
[17:08:44]: ▸ Found conflicting type declarations with the same name, which may indicate a build issue or a bug in Jazzy: unexposed , unexposed , unexposed , unexposed , unexposed , unexposed , unexposed , unexposed , unexposed
[17:08:44]: ▸ building site
[17:08:45]: ▸ building search index
[17:08:45]: ▸ 73% documentation coverage with 37 undocumented symbols
[17:08:45]: ▸ included 138 symbols
[17:08:46]: ▸ jam out ♪♫ to your fresh new docs in `docs`
```

Thoughts? Where do I look.

**Comment by johnfairh** [comment]
Ruby 2.4.1 should be fine.  This is an objective-C compile error though:
> [17:08:17]: ▸ /Users/amudubagilu/project-ios/Project/ProjectUmbrella.h:8:9: fatal error: could not build module ‘UIKit’

That and the `NS_ENUM` messages might suggest some difference in Xcode/SDK installation/`xcode-select` setting between the systems.

If that doesn't get you anywhere then if you share your jazzy config I can show you the clang command that looks to be failing.

**Comment by akac** [comment]
I figured out what was wrong. Our team didn't understand one aspect of Jazzy - it is using our built-framework and not the project files.

We could never get it to work (even after hours of trying, and literally dozens of different config files and choices) directly on the project, but once we pointed it to a pre-built framework - it worked fine.

So I had the pre-built framework and my team-mate didn't. That's why we were so confused as to why it wasn't working.
